forester/src/epoch_manager.rs

```diff
@@ -541,11 +527,24 @@ impl<R: RpcConnection, I: Indexer<R>> EpochManager<R, I> {
                     epoch_pda.epoch,
                 )
                 .await?;
+
+                // Prometheus metrics
+                let chunk_duration = start_time.elapsed();
+                queue_metric_update(epoch_info.epoch, num_tx_sent, chunk_duration).await;
+
+                // TODO: consider do we really need WorkReport
+                self.increment_processed_items_count(epoch_info.epoch, num_tx_sent)
```

Comment:
I see that we could use it to push things to graphana but in once we have multiple foresters by different parties we should get the performed work from the `ForesterEpochPda`s and the total performed work from the `EpochPda` .